.circleci/config.yml

```diff
@@ -392,17 +393,13 @@ jobs:
 
   # This job is manually triggered for now. see .circleci/README.md
   test-content-server-remote:
-    parameters:
-      executor:
-        type: executor
-    executor: << parameters.executor >>
+    executor: smoke-test-executor
```

Comment:
We don't actually need to parameterize this anymore. This was an artifact left over from testing when we were testing the performance differences of different instance sizes.